compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFinder.java

```diff
@@ -1152,6 +1152,10 @@ public void visit(BLangConstrainedType constrainedType) {
     public void visit(BLangStreamType streamType) {
         lookupNode(streamType.constraint);
         lookupNode(streamType.error);
+
+        if (symbolAtCursor == null) {
+            this.symbolAtCursor = streamType.type.type.tsymbol;
+        }
```

Comment:
@pubudu91 If this change required for stream type, then similar change also requires to table. Isn't it?

Reply:
Yeah, earlier there was an issue with supporting this. Hence why I originally didn't add it. Had another go at it and seems to be working now 🤔 So added with https://github.com/ballerina-platform/ballerina-lang/pull/30256/commits/7f7b3a6c482574008f690f52859c31d38e65ab15